client/pages/ListBikes.tsx

```diff
@@ -1,15 +1,23 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { View, StyleSheet, FlatList} from 'react-native';
 import BikeItem from '../components/BikeItem';
-import { bikes } from '../data/testBikes';
+import { getBikes } from "../services/bikes";
 
 
 const ListBikes = ({navigation}) => {
+    const [bikeArray, setBikeArray] = useState([]);
+
+    useEffect(() => {
+        getBikes()
+            .then(data => setBikeArray(data))
+            .catch(err => console.log(err));
+    }, []);
+
     return (
         <View style={styles.bikesContainer}>
             <FlatList style={styles.bikesWrapper}
                 keyExtractor={item => item.bike_id}
```

Comment:
This doesn't have anything to do with what you've done here, but I was getting Warning: Each child in a list should have a unique key here. Turns out, in our db, the bike document 'weojXdyHeQ24VBYjzPRO' didn't have a bike_id. I've added it in, but essentially when bike_id is undefined, the keyExtractor can't pull item.bike_id.